worlds/rabi_ribi/client/memory_io.py

```diff
@@ -32,6 +32,10 @@
 OFFSET_IN_WARP_MENU = int(0x16E5BB8)
 OFFSET_IN_COSTUME_MENU = int(0x16E6B20)
 OFFSET_CURRENT_WARP_ID = int(0x016E6D08)
+ # returns a memory address where various state is stored
+OFFSET_CURRENT_HEALTH_A = int(0x1682364)
```

Comment:
We should rename this to avoid needing a comment. From previous debug sessions, I think the other player state data is here, like the current X and Y coordinates (at `0x14` and `0x18` IIRC)

Consider something like `OFFSET_PLAYER_STATE` and `OFFSET_PLAYER_STATE_HEALTH`.